### PR TITLE
 Add logging configuration from file.

### PR DESCRIPTION
**Pull request by cosletmihai**

### CODE REVIEW

File: logging_config.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "format": "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+    }
+  },
+  "handlers": {
+    "default_logger": {
+      "class": "logging.StreamHandler",
+      "formatter": "default_formatter"
+    }
+  },
+  "loggers": {
```
Comment (costezki) [inline logging_config.json:14]: I would prefer to avoid a per-module definition. 
A simple project level config should be used instead.

**Review by costezki — CHANGES_REQUESTED** [review]
please define a logger configuration at the project level instead of module level.

File: logging_config.json (hunk 1 above)
Comment (cosletmihai) [reply logging_config.json:14]: project level config doesn't work as expected. we couldn't find a better way to define the logger

**Review by costezki — APPROVED** [review]
This solution looks close to what might the final one.

File: fingerprint/entrypoints/cli/main.py [hunk 2]
```diff
@@ -7,22 +7,14 @@
...
 
-__logger = logging.getLogger(__name__)
-
-
-# def copy_static_content(from_path, to_path):
-#     if pathlib.Path(from_path).is_dir():
-#         copy_tree(from_path, to_path)
-#     else:
-#         __logger.warning(from_path + " is not a directory !")
+logger = logging.getLogger('default')
```
Comment (costezki) [inline fingerprint/entrypoints/cli/main.py:17]: I would say let's use the root logger. 

```Return a logger with the specified name or, if the name is None, return a logger which is the root logger of the hierarchy.```

File: fingerprint/service_layer/handlers.py [hunk 3]
```diff
@@ -20,11 +20,11 @@
     import importlib_resources as pkg_resources
 import fingerprint_report_templates
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('default')
```
Comment (costezki) [inline fingerprint/service_layer/handlers.py:23]: same here, not sure we need a specially named logger

File: logging.conf [hunk 4]
```diff
@@ -0,0 +1,27 @@
+[loggers]
```
Comment (costezki) [inline logging.conf:1]: I guess we could get rid of root altogether; unless there are strong counter-indications

**Review by costezki — APPROVED** [review]

**Pull request merged**